### ISSUE TITLE
RankingManager not updating for Trackmania

### ISSUE THREAD

**Issue by agiorla** [ManiaControl/ManiaControl#208]
The RankingManager does not update the current ranking when playing in Trackmania. This is probably due to the fact that the Trackmania callbacks are not handled in the switch loop in the handleCallbacks function. 

I am currently trying to provide a fix.

**Comment by kremsy** [comment]
Yeah was developed for shootmania back days, i think its not even used anymore. Feel free to update it that it works with trackmania. Thanks!

**Comment by axelalex2** [comment]
@kremsy "I think its not even used anymore". You mean also in ShootMania? Because if so, we could clean it up. No need to keep something in ManiaControl, which is completely unused.

**Comment by kremsy** [comment]
Tbh im not 100% sure but I think its used nowhere and it is not working. We just kept it if plugins are associted with it. But if i remember right we marked it as deprecated, so it can be removed.
